src/cookieBanner/IStorage.ts

```diff
@@ -0,0 +1,5 @@
+export interface IStorage {
```

Comment:
Why is `IStorage` inside the cookieBanner folder?

Reply:
Because it is not used anywhere else yet.

Reply:
Moved.